contrib/cirrus/runner.sh

```diff
@@ -119,7 +119,12 @@ _run_integration() {
     # Ensure we start with a clean-slate
     podman system reset --force
 
+    # uncomment when sigstore integration tests merge
+    # make sigstore-testenv-up
```

Comment:
I think this should be uncommented already to make sure that it's working and that it will continue to work in the future.

Reply:
uncommented, thanks